### ISSUE TITLE
Data Request Service

### ISSUE THREAD

**Issue by SvenMokveldje** [GEWIS/sudosos-backend#354]
### What would you like?

To be able to return all personal data from a user, preferable as PDF and filtered such that no other users data is present.

### Why is this needed?

To make it easier for the GDPR officers if a GDPR request comes in.

**Comment by Yoronex** [comment]
I already did some work for this in the `feature/personal-data-extraction` branch.

**Comment by JustSamuel** [comment]
all personal data in a pdf? thats quite a lot.

Also PDFs SUCK. how about a textfile?

**Comment by Yoronex** [comment]
Agree with Samuel: it would be better to return a huge JSON containing all data, as returning this data in human-readable format is in my opinion not SudoSOS's responsibility. Other GEWIS services have the same problem, so it is in my opinion better to create a single, centralized solution for this.

**Comment by SvenMokveldje** [comment]
Yes, but could you not make the JSON file immediately as a PDF file that way it's still ugly but the GDPR officers can immediately stripe out other sensitive info

**Comment by JustSamuel** [comment]
> Yes, but could you not make the JSON file immediately as a PDF file that way it's still ugly but the GDPR officers can immediately stripe out other sensitive info

...like its a word document that you print to pdf?
